closed/autoconf/custom-hook.m4

```diff
@@ -152,15 +152,22 @@ AC_DEFUN([OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU],
     powerpc64)
       OPENJ9_CPU=ppc-64
       ;;
+    arm)
+      OPENJ9_CPU=arm
+      ;;
```

Comment:
Changes to autoconf files are expected to be accompanied by regenerated configure scripts.

Reply:
OK, I wasn't sure if I was allowed to check in the generated scripts. I can add those.

Reply:
Scripts regenerated and added to the PR